### PR TITLE
Editorial: Update whatwg/meta references in README.md

### PR DESCRIPTION
**Pull request by snuggs**
References https://github.com/whatwg/meta/issues/37

***
[Preview](https://s3.amazonaws.com/pr-preview/snuggs/infra/patch-1.html) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/infra/a3a49bc...snuggs:4e45b6a.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ Folks notice minor and larger issues with the Infra Standard all the time and we
 your help fixing those. Pull requests for typographical and grammar errors are also most welcome.
 
 We'd be happy to mentor you through this process. If you're interested and need help getting
-started, leave a comment on the issue or ask around [on IRC](https://wiki.whatwg.org/wiki/IRC).
+started, review [WHATWG contributing guidelines](https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md),
```
Comment (annevk) [inline README.md]: Missing "the" before the link.

**Comment by annevk** [comment]
The Merge policy section can also be updated to point directly to the document in whatwg/meta.

File: README.md (hunk 1 above)
Comment (snuggs) [reply README.md]: Fixed in https://github.com/whatwg/infra/pull/170/commits/2ecd75666a6b6fb485d65b0890f52f4ad7657228

**Comment by snuggs** [comment]
@annevk Fixed in https://github.com/whatwg/infra/commit/2ecd75666a6b6fb485d65b0890f52f4ad7657228 & https://github.com/whatwg/infra/pull/170/commits/4e45b6ad2c2cfac72cd05a526b610c43ef526e1d

**Comment by annevk** [comment]
🎉

**Pull request merged**